examples/datasets/plot_spm_faces_dataset.py

```diff
@@ -5,13 +5,14 @@
 ==========================================
```

Comment:
It was an issue with the `random_state` of `ICA` interacting with the component removal below, next commit should fix it